TEI.js

```diff
@@ -196,6 +223,9 @@ function generateItem(item, teiDoc) {
             bibl.setAttributeNS(ns.xml, "xml:id", myXmlid);
             exportedXMLIds[myXmlid] = bibl;
         }
+        //create attribute for Zotero item URI
+        var itemURI = genURI(item);
+        bibl.setAttribute("corresp", itemURI);
```

Comment:
you don't need to assign to variables first. `bibl.setAttribute("corresp", item.uri);` would be sufficient.